Every other build of clr component fails due to Microsoft.NET.Sdk.IL.targets not found

Every other build of clr component (first is ok, second fails, third is ok, fourth fails etc). with the following error

D:\git\runtime6\.dotnet\sdk\10.0.100-rc.1.25420.111\Sdks\NuGet.Build.Tasks.Pack\build\NuGet.Build.Tasks.Pack.targets(221,5): error NU5019: File not found: 'D:\git\runtime6\artifacts\obj\coreclr\Microsoft.NET.Sdk.IL\windows.x64.Debug\Microsoft.NET.Sdk.IL.targets'. [D:\git\runtime6\src\coreclr\.nuget\Microsoft.NET.Sdk.IL\Microsoft.NET.Sdk.IL.proj]

My build command is e.g. `.\build.cmd clr+libs -c Release -rc Checked`

This happens both on Windows and macOS (I haven't tried others)

--- a/src/coreclr/.nuget/Microsoft.NET.Sdk.IL/Microsoft.NET.Sdk.IL.proj
+++ b/src/coreclr/.nuget/Microsoft.NET.Sdk.IL/Microsoft.NET.Sdk.IL.proj
@@ -27,9 +27,7 @@
 
     <GenerateFileFromTemplate TemplateFile="$(ILTargetsTemplateFile)"
                               OutputPath="$(ILTargetsOutputFile)"
-                              Properties="IlAsmVersion=$(Version)">
-      <Output TaskParameter="ResolvedOutputPath" ItemName="FileWrites" />
-    </GenerateFileFromTemplate>
+                              Properties="IlAsmVersion=$(Version)" />
 
     <ItemGroup>
       <None Include="$(ILTargetsOutputFile)" Pack="true" PackagePath="targets/" />

